frontend/src/components/problem/ProblemDisplay.tsx

```diff
@@ -271,8 +271,8 @@ function ProblemDisplay(props: ProblemDisplayParams) {
           </div>
         ) : null}
 
-      <LargeInputButton value={actionText} onClick={() => onClick(newProblem)} />
-      {editMode ? <LargeInputButton value="Delete Problem" onClick={deleteProblemFunc} /> : null}
+      <PrimaryInput value={actionText} onClick={() => onClick(newProblem)} />
+      {editMode ? <PrimaryInput value="Delete Problem" onClick={deleteProblemFunc} /> : null}
```

Comment:
This is a good change, thanks! Just wanted to note that you'll run into merge conflicts I think, so this may have to be updated

Reply:
Just fixed - I think I could just delete this refactor since the buttons not being used there anymore anyways